src/style.css

```diff
@@ -1 +1,62 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc);
+}
+
+input {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+}
+
+.barra-big {
```

Comment:
It is better to use the name in full English "barra" is a Portuguese word.
Also, "small" is a modifier, and therefore should be separated by 2 hyphens instead of 1. Like this: `.bar--big`
Same advice is valid for the other class "barra-small"